x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/select_rule_type/index.tsx

```diff
@@ -5,19 +5,58 @@
  */
 
 import React, { useCallback } from 'react';
-import { EuiCard, EuiFlexGrid, EuiFlexItem, EuiIcon, EuiFormRow } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import {
+  EuiCard,
+  EuiFlexGrid,
+  EuiFlexItem,
+  EuiFormRow,
+  EuiIcon,
+  EuiLink,
+  EuiText,
+} from '@elastic/eui';
 
 import { FieldHook } from '../../../../../shared_imports';
 import { RuleType } from '../../../../../containers/detection_engine/rules/types';
 import * as i18n from './translations';
 import { isMlRule } from '../../helpers';
 
+const MlCardDescription = ({ hasValidLicense = false }: { hasValidLicense: boolean }) => (
```

Comment:
Do you want the `?` as in:

```ts
const MlCardDescription = ({ hasValidLicense = false }: { hasValidLicense?: boolean }) => (
```

Since you have the ability to do a default of false?